recipe/bld.bat

```diff
@@ -19,21 +19,42 @@ if "%ARCH%"=="32" (
     set PLATFORM=x64
 )
 
-
+REM build static library
 msbuild.exe /p:Platform=%PLATFORM% /p:Configuration=Release lib_mpir_gc\lib_mpir_gc.vcxproj
 msbuild.exe /p:Platform=%PLATFORM% /p:Configuration=Release lib_mpir_cxx\lib_mpir_cxx.vcxproj
+REM build dll library
+msbuild.exe /p:Platform=%PLATFORM% /p:Configuration=Release dll_mpir_gc\dll_mpir_gc.vcxproj
```

Comment:
Thanks for clarification. Might be good to note this in comment  for dll build, just to answer that question in the future. 

Otherwise, +1 to merge from me. (I am not on this team and can't merge this myself.)